ci/rpm/build_unsuccessful.sh

```diff
@@ -24,3 +25,7 @@ fi
 (if cd "$mockroot/result/"; then
   cp -r . "$artdir"
 fi)
+
+if ls $mockroot/root/builddir/build/BUILD/daos-*/config"${ARCH}".log; then
+    mv $mockroot/root/builddir/build/BUILD/daos-*/config"${ARCH}".log ${artdir}/config.log-rpm
```

Comment:
(style)  line over 80 characters
(lint) Double quote to prevent globbing and word splitting. [SC2086]
(lint) Double quote to prevent globbing and word splitting. [SC2086]